package.json

```diff
@@ -29,9 +29,8 @@
   "dependencies": {
     "@embroider/macros": "^1.6.0",
     "@embroider/util": "^1.6.0",
-    "@lblod/submission-form-helpers": "^1.4.1",
+    "@lblod/submission-form-helpers": "git@github.com:lblod/submission-form-helpers.git#9a84dc13cb36c902d28f9e35b3822e8e98781239",
```

Comment:
Using a specific commit until the branch is merged and released. Both `@lblod/submission-form-helpers` and `forking-store` use rdflib internally so we use the branch where the browser-rdflib to rdflib 2 conversion happened.